std/php/_std/Array.hx

```diff
@@ -273,3 +298,12 @@ private extern interface ArrayAccess<K, V> {
 	private function offsetSet(offset:K, value:V):Void;
 	private function offsetUnset(offset:K):Void;
 }
+
+@:native('Iterator')
+private extern interface NativeIterator<K, V> {
```

Comment:
This should be moved to an own module in `php` package similar to `php.ArrayAccess`. It could be useful in other cases